docs/models.md

```diff
@@ -95,45 +97,51 @@ book.isbn # => RFC4122 V4 UUID string
 ```
 ## Default values
 
-A default value can be assigned to a field that will be used if another value is not specified/supplies.
+A default value can be defined that will be used if another value is not specified/supplied.
 
 ```Crystal
 class Book < Granite::Base
-  adapter mysql
-  
-  field name : String, default: "DefaultBook"
+  connection mysql
+
+  column id : Int64, primary: true
+  column name : String = "DefaultBook"
 end
 
 book = Book.new
 book.name # => "DefaultBook"
 ```
 
-
 ## Generating Documentation
 
-By default, running `crystal docs` will **not** include Granite methods, constants, and properties.  To include these, have an ENV variable: `DISABLE_GRANITE_DOCS=false` set before running `crystal docs`.
+By default, running `crystal docs` will **not** include Granite methods, constants, and properties.  To include these, use the `granite_docs` flag when generating the documentation.  E.x. `crystal docs -D granite_docs`.
 
-The `field` and `primary` macros have a comment option that will specify the documentation comment to apply to that property's getter and setter.
+Doc block comments can be applied above the `column` macro.
 
-`field age : Int32, comment: "# Number of seconds since the post was posted"`
+```crystal
+# If the item is public.
+column published : Bool
+```
 
-## Third-Party Annotations
+## Annotations
 
-Annotations can be a powerful method of adding property specific features with minimal amounts of code.  Since Granite utilizes the `property` keyword for its fields, annotations are able to be applied easily.  This is accomplished by using the `annotations` option on a field, similar to the `comment` option above.  It is used in the form of `annotations: ["annotation1", "annotation2"]`, for example:
+Annotations can be a powerful method of adding property specific features with minimal amounts of code.  Since Granite utilizes the `property` keyword for its columns, annotations are able to be applied easily.  These can either be `JSON::Field`, `YAML::Field`, or third party annotations.
 
 ```Crystal
 class Foo < Granite::Base
-  adapter mysql
-  table_name foos
+  connection mysql
+  table foos
+
+  column id : Int64, primary: true
+  
+  @[JSON::Field(ignore: true)]
+  @[Bar::Settings(other_option: 7)]
+  column password : String
 
-  field name : String
-  field password : String, annotations: ["@[Foo::Options(option1: true)]", "@[Bar::Settings(other_option: 7)]"]
-  field age : Int32
+  column name : String
+  column age : Int32
 end
 ```
```

Comment:
Do you want to document the new annotations that are available in Granite?

Reply:
I think I would rather them be an implementation detail for the moment.